renderer/support/build.gradle

```diff
@@ -0,0 +1,45 @@
+apply plugin: 'com.android.library'
```

Comment:
why need renderer/support project?

Reply:
This is unnecessary and has been removed